spec/moronepo_activate_action_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe Fastlane::Actions::MoronepoActivateAction do
+  describe 'moronepo_activate' do
+    it 'should run proper shell command' do
```

Comment:
How does it handle errors? Is it straightforward for the user what went wrong?

Reply:
It print error to console and fail lane:
```
[11:57:02]: --------------------------
[11:57:02]: --- Step: moronepo_run ---
[11:57:02]: --------------------------
[11:57:02]: $ flutter pub global run moronepo --working-directory example_projects --filter hasTests run -- flutter test
[11:57:03]: ▸ Running "flutter test" for project_with_tests project
[11:57:03]: ▸ Project directory "example_projects/project_with_tests"
[11:57:05]: ▸ 00:01 +0 -1: loading /Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart [E]
[11:57:05]: ▸ Failed to load "/Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart":
[11:57:05]: ▸ Compilation failed
[11:57:05]: ▸ Test: /Users/marcin.s/Documents/Meniga/fastlane-plugin-moronepo/example_projects/project_with_tests/test/i_am_a_test.dart
[11:57:05]: ▸ Shell: /Users/marcin.s/flutter/bin/cache/artifacts/engine/darwin-x64/flutter_tester
[11:57:05]: ▸ 00:01 +0 -1: Some tests failed.
[11:57:05]: ▸ Compiler message:
[11:57:05]: ▸ Failed to load test harness. Are you missing a dependency on flutter_test?
[11:57:05]: ▸ pub finished with exit code 1
```